apps/explorer/hooks/useFaucetFund.ts

```diff
@@ -24,6 +24,6 @@ type FaucetFundResponse = {
 export function useFaucetFund() {
   return usePostFunc<undefined, FaucetFundPayload, FaucetFundResponse>({
     api: faucetApi,
-    route: '/request',
+    route: '/',
```

Comment:
@n8maninger is fund at  `/zen/faucet` and status at `/zen/faucet/:id`?

Reply:
It was at `/request` and `/request/:id`, but it makes more sense to drop the prefix entirely